### ISSUE TITLE
JSON flag ignored under certain circumstances

### ISSUE THREAD

**Issue by Theowningone** [SkaceKamen/sqflint#1]
Not entirely sure what's causing this, initially I thought it was related to multiple errors, but I now believe it has something specific to do with the pattern.

When linting this code, with `--json` passed to sqflint.exe, it returns the result in text, and not in JSON.  If `<>;` is removed from line 1, the result is properly returned in JSON.
#### Code (testfile.sqf):

``` sqf
        _caller = _this select 1;<>;
        _nearstuff = nearestobjects [_lc, ["landvehicle", "camanbase"], 8];qsdasrfd1234adf;'l[;a.d;[sd,c[opsalfksf'
        .]';.'/d'a.d';asd['asdsd;a;sdasd;a[s/';;'d'asd/]asd
        _attachedObjs = [];
```
#### Result 1 (with `<>;`):

```
PS C:\sqflint> .\sqflint.exe --json .\testfile.sqf
Encountered " "<" "< "" at line 1, column 28.
Was expecting one of:
    <EOF>
    "!" ...
    "not " ...
    "for" ...
    "if" ...
    "objNull" ...
    "switch" ...
    "while" ...
    "true" ...
    "false" ...
    <INTEGER_LITERAL> ...
    <FLOATING_POINT_LITERAL> ...
    <STRING_LITERAL> ...
    <STRING_LITERAL_OTHER> ...
    <IDENTIFIER> ...
    "(" ...
    "{" ...
    "[" ...
    ";" ...
```
#### Result 2 (without `<>;`):

```
PS C:\sqflint> .\sqflint.exe --json .\testfile.sqf
{"line":[3,3],"column":[3,3],"type":"error","message":"Encountered \".\".\r\nWas expecting one of:\r\n    \"=\" ...\r\n    \">\" ...\r\n    \"<\" ...\r\n    \">=\" ...\r\n    \"<=\" ...\r\n    \"!\" ...\r\n    \"not \" ...\r\n    \"==\" ...\r\n    \"||\" ...\r\n    \"&&\" ...\r\n    \"+\" ...\r\n    \"-\" ...\r\n    \"*\" ...\r\n    \"/\" ...\r\n    \"^\" ...\r\n    \"%\" ...\r\n    \"mod\" ...\r\n    \">>\" ...\r\n    \"objNull\" ...\r\n    \"true\" ...\r\n    \"false\" ...\r\n    <INTEGER_LITERAL> ...\r\n    <FLOATING_POINT_LITERAL> ...\r\n    <STRING_LITERAL> ...\r\n    <STRING_LITERAL_OTHER> ...\r\n    <IDENTIFIER> ...\r\n    \"(\" ...\r\n    \"{\" ...\r\n    \"[\" ...\r\n    \";\" ...\r\n    \"!=\" ...\r\n    "}
{"line":[2,2],"column":[32,34],"type":"warning","message":"Possibly undefined variable _lc"}
```

**Comment by SkaceKamen** [comment]
Thanks for reporting, I'll look at it asap.

**Comment by SkaceKamen** [comment]
Fixed in [v0.5.1](https://github.com/SkaceKamen/sqflint/releases/tag/v0.5.1) release.
